AutomatedTesting/Gem/PythonTests/scripting/NewScriptEventButton_HappyPath_ContainsSCCategory.py

```diff
@@ -19,10 +19,6 @@ class Tests:
     new_asset = "Asset Editor created with new asset"
     script_event = "New Script event created in Asset Editor"
```

Comment:
Previous tests have used the the `var = ('pass condition message', 'fail condition message')`, why is that pattern not being used here?